model/test_results.go

```diff
@@ -88,6 +90,11 @@ func (t *TestResults) Setup(e cedar.Environment) { t.env = e }
 // IsNil returns if the TestResults is populated or not.
 func (t *TestResults) IsNil() bool { return !t.populated }
 
+// PrestoPartitionKey returns the partition key for the S3 bucket in Presto.
+func (t *TestResults) PrestoPartitionKey() string {
+	return fmt.Sprintf("task_create_iso=%s/project=%s/%s", t.CreatedAt.UTC().Format("2006-01-02"), t.Info.Project, t.ID)
```

Comment:
Can the date format be a constant?

Are both the CLI dates and the partition key making the same assumption about day-level timestamp granularity in Presto? If so, should they use the same shared date format?

Reply:
Done. I made it a private global variable because we are going to delete the CLI code anyways.